pkg/model/core/grpcroute.go

```diff
@@ -79,6 +79,13 @@ func (r *GRPCRoute) Inner() *gwv1alpha2.GRPCRoute {
 	return &r.r
 }
 
+func (r *GRPCRoute) GK() metav1.GroupKind {
```

Comment:
Why GK() instead of GroupKind()?

Reply:
its GK or GKV in other places of kubernetes: apimachinery for example

Reply:
I am only able to find examples of GroupKind(), such as [here](https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/builder/controller.go#L367C17-L367C17)

Reply:
https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.16.3/pkg/client/apiutil

if it bothers you I can change